packages/ui/src/components/ProgressBar/__stories__/Label.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import type { StoryFn } from '@storybook/react'
+import { ProgressBar } from '..'
+import { Badge } from '../../Badge'
+import { Stack } from '../../Stack'
+
+export const Label: StoryFn = props => (
+  <Stack gap={3}>
+    <ProgressBar {...props} direction="row" />
+    <ProgressBar {...props} direction="column" />
+  </Stack>
+)
```

Comment:
this could be in an individual story, we would have label and labelDescription explanation and appart, direction. What do you think?